conf/logstash/99-firefly-fwd.conf.example

```diff
@@ -0,0 +1,10 @@
+output {
+  udp {
+    host => "<REPLACE_WFIREFLY_COLLECTOR>"
+    id => "udp_out"
+    port => 10514
+    codec => plain {
+      format => "%{message}"
```

Comment:
This could probably work also based on [@metadata][syslog_msg] field (in which case we don't need to whitelist "message" in prune)